torch/nn/modules/dropout.py

```diff
@@ -21,22 +24,25 @@ class Dropout(Module):
         >>> output = m(input)
     """
 
-    def __init__(self, p=0.5, inplace=False):
+    def __init__(self, p=0.5, inplace=False, scale_train=True):
         super(Dropout, self).__init__()
         if p < 0 or p > 1:
             raise ValueError("dropout probability has to be between 0 and 1, "
                              "but got {}".format(p))
         self.p = p
         self.inplace = inplace
+        self.scale_train = scale_train
 
     def forward(self, input):
```

Comment:
Do you mean have something like this?
`def __setstate__(self, state):
     if 'scale_train' not in state:
         self.__dict__.update({'scale_train' : True})`